appveyor.yml

```diff
@@ -0,0 +1,32 @@
+environment:
+  global:
+    RUST_BACKTRACE: 1
+    Features: "use-mock-routing"
+  matrix:
+    - RUST_VERSION: stable
+#    - RUST_VERSION: beta
+#    - RUST_VERSION: nightly
+branches:
+  only:
+    - master
+
+clone_depth: 50
+
+install:
+  - ps: |
+        $url = "https://github.com/maidsafe/QA/raw/master/Powershell%20Scripts/AppVeyor"
+        cmd /c curl -fsSL -o "Install Rustup.ps1" "$url/Install%20Rustup.ps1"
+        cmd /c curl -fsSL -o "Build.ps1" "$url/Build.ps1"
+        cmd /c curl -fsSL -o "Run Tests.ps1" "$url/Run%20Tests.ps1"
+        . ".\Install Rustup.ps1"
+
+platform:
+  - x86
+  - x64
+
+configuration:
+#  - Debug
+  - Release
+
+test_script:
+  - ps: . "cargo run --example test"
```

Comment:
We don't need powershell here ?
Basically base it on safe_core as the previous comments say.